lib/constable_web/templates/announcement/show.html.eex

```diff
@@ -46,27 +46,27 @@
           <%= relative_timestamp(@announcement.inserted_at) %>
         </div>
       </div>
-      <div class="subscription">
-        <%= if @subscription do %>
-          <%= link to: Routes.announcement_subscription_path(@conn, :delete, @announcement),
-            method: :delete,
-            class: "tbds-button button-secondary" do %>
-            <svg class="tbds-button__icon tbds-button__icon--start">
-              <use xlink:href="/images/icons.svg#check"></use>
-            </svg>
-            <%= gettext("Subscribed to thread") %>
-          <% end %>
-        <% else %>
-          <%= link to: Routes.announcement_subscription_path(@conn, :create, @announcement),
-            method: :post,
-            class: "tbds-button button-secondary" do %>
-            <svg class="tbds-button__icon tbds-button__icon--start">
-              <use xlink:href="/images/icons.svg#plus"></use>
-            </svg>
-            <%= gettext("Subscribe to thread") %>
-          <% end %>
+    </div>
+    <div class="subscription">
+      <%= if @subscription do %>
+        <%= link to: Routes.announcement_subscription_path(@conn, :delete, @announcement),
+          method: :delete,
+          class: "tbds-button button-secondary" do %>
+          <svg class="tbds-button__icon tbds-button__icon--start">
+            <use xlink:href="/images/icons.svg#check"></use>
+          </svg>
+          <%= gettext("Subscribed to thread") %>
         <% end %>
-      </div>
+      <% else %>
+        <%= link to: Routes.announcement_subscription_path(@conn, :create, @announcement),
+          method: :post,
+          class: "tbds-button button-secondary" do %>
+          <svg class="tbds-button__icon tbds-button__icon--start">
+            <use xlink:href="/images/icons.svg#plus"></use>
+          </svg>
+          <%= gettext("Subscribe to thread") %>
```

Comment:
I know this was existing and not change in this PR - but it’s weird we suddenly start using the term “thread” instead of “announcement.”